### PR TITLE
:arrow_up: Update courage to 2.1.0

### PR DESCRIPTION
**Pull request by jhe-okta**
- Also updated install-with-shrinkwrap to 1.6.0 (does not run dedupe)

Resolves: OKTA-119240
Bacon: test

@ukilon-okta @mauriciocastillosilva-okta @rchild-okta @alexeystadnikov-okta  please review.

### CODE REVIEW

File: node_modules/@okta/courage/package.json [hunk 1]
```diff
@@ -1,54 +1,59 @@
...
+      "@okta/courage@2.1.0",
+      "/home/vagrant/okta/okta-signin-widget"
     ]
   ],
-  "_from": "@okta/courage@1.17.0",
-  "_id": "@okta/courage@1.17.0",
+  "_from": "@okta/courage@2.1.0",
+  "_id": "@okta/courage@2.1.0",
   "_inCache": true,
+  "_installable": true,
```
Comment (mauriciocastillosilva-okta) [inline node_modules/@okta/courage/package.json]: this can be removed

File: node_modules/@okta/courage/package.json [hunk 2]
```diff
@@ -1,54 +1,59 @@
...
+    "raw": "@okta/courage@2.1.0",
+    "rawSpec": "2.1.0",
     "scope": "@okta",
-    "spec": "1.17.0",
+    "spec": "2.1.0",
     "type": "version"
   },
   "_requiredBy": [
-    "/"
+    "#DEV:/"
```
Comment (mauriciocastillosilva-okta) [inline node_modules/@okta/courage/package.json]: this probably also should be ```"/"```

**Comment by jhe-okta** [comment]
@mauriciocastillosilva-okta I've done what you suggested in the two comments in the last commit. Maybe you comment from the previous changes?

**Comment by mauriciocastillosilva-okta** [comment]
@jhe-okta you did it for ```@okta/i18n/package.json``` not for ```@okta/courage/package.json```

**Comment by jhe-okta** [comment]
@mauriciocastillosilva-okta got it. Please look again.

**Comment by jhe-okta** [comment]
@mauriciocastillosilva-okta with the courage package.json file change, security key (u2f) test failed.

**Pull request merged**